cores/arduino/USBAPI.h

```diff
@@ -196,6 +196,7 @@ bool	CDC_Setup(USBSetup& setup);
 int USB_SendControl(uint8_t flags, const void* d, int len);
 int USB_RecvControl(void* d, int len);
 int USB_RecvControlLong(void* d, int len);
+bool USB_SendStringDescriptor(const u8*, u8, uint8_t);
 
```

Comment:
This function already defined in the USBCore.cpp so making it public seems logical. The functuin can be used in the scenario when the text value needs to be passed from the device to the host. Examples - serial number, OEM Vendor, battery chemistry etc.

Reply:
this is also a deviation from the USBAPI as defined in the Arduino Core API https://github.com/arduino/ArduinoCore-API/blob/master/api/USBAPI.h

Reply:
> 
> 
> this is also a deviation from the USBAPI as defined in the Arduino Core API https://github.com/arduino/ArduinoCore-API/blob/master/api/USBAPI.h

I believe its ok since it is just an exposure of the existing method,  and is inline with overall architecture. Ability of sending string descriptors is an essential part of USB protocol, I believe it will be useful for many projects.

Reply:
I opened an issue in the arduino core api project, related to the implementation of the USBAPI https://github.com/arduino/ArduinoCore-API/issues/141